client/blocks/upload-image/index.jsx

```diff
@@ -157,6 +272,14 @@ class UploadImage extends Component {
 			);
 		}
 
+		if ( typeof uploadingDoneContent === 'undefined' && uploadedImage && uploadedImage.URL ) {
+			uploadingDoneContent = (
+				<div className="upload-image__uploading-done-container">
+					<img src={ uploadedImage.URL } />
+				</div>
+			);
```

Comment:
But isn't it the same? If the condition is not `true`, the `*Content` elements won't be created. Even if I put them to a separate function, that function would be parsed by JS whether it's used or not.

Or am I wrong?